gnome/gstreamer1-gst-libav/Portfile

```diff
@@ -22,17 +23,24 @@ master_sites        https://gstreamer.freedesktop.org/src/${my_name}/
 distname            ${my_name}-${version}
 use_xz              yes
 
-checksums           rmd160  fb8bdd29adba833ea8f2c9e618828ffdf19b82c8 \
-                    sha256  c724f612700c15a933c7356fbeabb0bb9571fb5538f8b1b54d4d2d94188deef2 \
-                    size    9466288
+checksums           rmd160  64f7a8e2a217c4fff7dc9544de8a0f462e025ea8 \
+                    sha256  42f93f5ce9a3fc22051e5f783a4574b56ebf213f331f75dcbc3552459bd3a06a \
+                    size    272124
 
-depends_build       port:pkgconfig \
-                    port:nasm
+depends_build-append        port:python38 \
+                            port:pkgconfig \
+                            port:nasm
 
-depends_lib         port:gstreamer1-gst-plugins-base
+depends_lib-append          port:gobject-introspection \
+                            port:gstreamer1-gst-plugins-base \
+                            port:ffmpeg
 
-# This will cause the installed ffmpeg headers to be used, remove it.
-configure.cppflags-delete -I${prefix}/include
+# meson.build uses python3 for multiple checks
+pre-configure {
+    file mkdir ${workpath}/bin
+    file link -symbolic ${workpath}/bin/python3 ${prefix}/bin/python3.8
```

Comment:
Note that MacPorts has an `ln` procedure that works like the `ln` executable so you could write:
```
ln -s ${prefix}/bin/python3.8 ${workpath}/bin/python3
```

Reply:
I wasn't sure the more "macports" correct way as some Portfiles did this some used ln, I'll swap this over to using `ln` and update the comment.